script/utils/check-nonce-overrides.sh

```diff
@@ -3,14 +3,14 @@
 search_dir="./tasks"
 
 # Find all .env files in the given directory and its subdirectories
-env_files=$(find "$search_dir" -type f -name "*.env")
+env_files=$(find "$search_dir" -type f -name '*.env')
 
 # Process each .env file
 for file in $env_files; do
     echo "Checking file: $file"
 
     # Process the file line by line
-    while IFS= read -r line || [[ -n "$line" ]]; do
```

Comment:
with this current fix we skip some line. 
I prefer to keep the previous version here.
I have run few tests and we miss approx 10 lines and 2 SAFE_NONCES line.
cf the line here: 
<img width="1676" alt="image" src="https://github.com/user-attachments/assets/2e253503-100d-4c3c-833b-129676643662" />